packages/cli/src/cmds/validator/options.ts

```diff
@@ -303,15 +304,16 @@ export const validatorOptions: CliCommandOptions<IValidatorCliArgs> = {
     type: "boolean",
   },
 
-  // Remote signer
+  // External signer
```

Comment:
it would be nice to have a section in the docs that described how to use these features or at least linked to the relevant external specs / docs

Reply:
Definitely good idea to have a dedicated docs page for that, noted that down in https://github.com/ChainSafe/lodestar/issues/5961#issuecomment-1723112851 a while ago. It seems that most people figure it out based on CLI docs, and for `--externalSigner.fetch`, fetching continuously is what should be the expected behavior anyways.

We should probably create an issue for each missing docs page we want to add in the future and the issues need to be detailed enough so anyone could potentially pick it up.

I can get this process started by adding an issue for the external signer page.

Reply:
Added external signer docs page to this PR in https://github.com/ChainSafe/lodestar/pull/6672/commits/836c87556ee9d2b6b2e5e285ba4e08f6a6effaac